templates/Procfile

```diff
@@ -1,2 +0,0 @@
-web: bundle exec puma -p $PORT -C ./config/puma.rb
-worker: bundle exec rake jobs:work
```

Comment:
@tute did you compare our config with the Rails default? There are some differences I saw right away:

* They default to a much higher number of threads
* They don't use multiple processes
* They use the `tmp_restart` plugin, which doesn't seem useful with Heroku/`heroku local`

Our total concurrency (2 threads * 2 workers) is similiar to theirs (5 threads * 1 worker), but I'm guessing we actually have more concurrency, as multiple Ruby threads will block each other on non-IO actions like rendering views.